pages/sponsorship.js

```diff
@@ -13,6 +15,18 @@ export default () => {
         {/* Call-to-action goes here */}
       </HeroBanner>
 
+      <Content
+        title="Become A Corporate Sponsor Today"
+        theme="white"
+        className={styles.justifyAlign}
+        columns={[
+          <p className="changeToSomething">
+            Operation Code is pleased to invite America’s leading technology companies to become
```

Comment:
can you use `&apos;` instead of `'`? You also have one on line 26 in `today's`

Reply:
yes!